### DISCUSSION TITLE
Setting suspend=y with jvm-debug in the bash script.

### DISCUSSION THREAD

**Discussion by dicklieber** [sbt/sbt-native-packager#1593] [Q&A]
I'm trying to figure put how to use _suspend=y_ instead of _suspend=n_ in the bash script.
I certainly can edit the bash script in the deployed application, but it seems like I should be able to configure this in by build.sbt,

It looks like I'd need to override the addDebugger () method and do my own -agentli... But I can't figure out how to do that in sbt configuration. It' probably trivial, but I can't figure out that from the sbt-native docs. That doesn't mean it's not there, It's just I can't :-)

I could completely replace the bash-template in my build.sbt, but that seems to be overkill.

Thanks for any suggestions

**Comment by dwickern** [comment]
Instead of `-jvm-debug`, you can pass your desired `-agentlib:...` argument to the script.
